components/expenses/ConfirmProcessExpenseModal.tsx

```diff
@@ -236,9 +242,19 @@ export default function ConfirmProcessExpenseModal({ type, onClose, expense }: C
     <StyledModal role="alertdialog" width="432px" onClose={onClose} trapFocus>
       <ModalHeader>{intl.formatMessage(MessagesPerType[type].title)}</ModalHeader>
       <ModalBody pt={2}>
-        <P mb={3} color="black.700" lineHeight="20px">
-          {intl.formatMessage(MessagesPerType[type].description)}
-        </P>
+        <Flex>
+          <AlertTriangleIcon size={30} color="orange" />
+          <P mb={3} color="black.700" lineHeight="20px" ml={10}>
+            {intl.formatMessage(MessagesPerType[type].description)}
+          </P>
+        </Flex>
```

Comment:
We don't have this pattern in our design system. If you want to highlight this message, I can recommend using a `<MessageBox type="warning" withIcon>` instead.